scripts/gulpfiles/pxt_tasks.js

```diff
@@ -42,36 +46,27 @@ function pxtPublishTsTask() {
 const pxtTest = gulp.series([buildTasks.core, pxtPublishTask]);
 
 // Task for bumping patch version and tagging commit. Travis will upload to npm.
-const pxtBump = gulp.series([
+const pxtBump = gulp.series(
   // Sync to latest
-  function (done) {
-    gulp.git.checkout('develop');
-    gulp.git.pull('origin',' develop');
-    done();
-  },
+  // execSync('git checkout develop', { stdio: 'inherit' }),
+  // execSync('git pull origin develop', { stdio: 'inherit' }),
   // Build compressed files and typings
   buildTasks.core,
   typings.typings,
-  // Increment version, tag and push
   function (done) {
     var v = semver.inc(JSON.parse(fs.readFileSync('./package.json', 'utf8')).version, 'patch');
     gulp.src('./package.json')
       .pipe(gulp.bump({ "version": v }))
-      .pipe(gulp.dest('./'));
-
-    gulp.src('.')
-      .pipe(gulp.git.add())
-      .pipe(gulp.git.commit(v))
+      .pipe(gulp.dest('./'))
       .on('end', function () {
-        gulp.git.tag('v' + v, v, function (error) {
-          if (error) {
-            return done(error);
-          }
-          gulp.git.push('origin', '', { args: '--tags' }, done);
-        })
+        execSync('git add .', { stdio: 'inherit' });
```

Comment:
this seems a little dangerous? I feel like bump should be an empty commit

Reply:
hmm so it bumps the version number in the package.json and rebuilds the compressed files and typings, but i could move that to be a separate action... or add a specific named set of files?

Reply:
yeah, i think a list would be good